modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/dashboard-remote-app/src/routes/deals-chart/index.js

```diff
@@ -113,27 +113,20 @@ export default function () {
 					['Submitted', ...leadsChartValues?.submitted],
 					['Approved', ...opportunitiesChartValues?.approved],
 					['Rejected', ...totalRejectedChartValues],
-					['Closed Won', ...opportunitiesChartValues?.closedWon],
 				],
-				groups: [['submitted', 'approved', 'closedwon']],
+				groups: [['submitted', 'approved']],
 				order: 'desc',
 				type: 'bar',
 				types: {
 					approved: 'bar',
-					closedwon: 'bar',
 					rejected: 'spline',
 					submitted: 'bar',
 				},
 				x: 'x',
 			},
 			grid: {
 				y: {
-					lines: [
-						{value: 100},
-						{value: 200},
-						{value: 300},
-						{value: 400},
-					],
+					lines: [{value: 100}, {value: 300}, {value: 400}],
```

Comment:
I believe these values has to be sequential, right? 
```suggestion
					lines: [{value: 100}, {value: 200}, {value: 300}],
```